addons/spreadsheet_dashboard/views/menu_views.xml

```diff
@@ -28,8 +28,15 @@
 
     <record id="spreadsheet_dashboard_action_configuration_dashboards" model="ir.actions.act_window">
         <field name="name">Dashboards</field>
-        <field name="res_model">spreadsheet.dashboard.group</field>
+        <field name="res_model">spreadsheet.dashboard</field>
         <field name="view_mode">list,form</field>
+        <field name="context">{"search_default_groupby_dashboard_group_id" : True}</field>
+    </record>
+
+    <record id="spreadsheet_dashboard_action_configuration_sections" model="ir.actions.act_window">
```

Comment:
according to [guidelines](https://www.odoo.com/documentation/19.0/contributing/development/coding_guidelines.html#xml-ids-and-naming)
```suggestion
    <record id="spreadsheet_dashboard_group_action_configuration_sections" model="ir.actions.act_window">
```